ripe/atlas/tools/commands/measure/http.py

```diff
@@ -50,7 +50,7 @@ def add_arguments(self):
         )
         specific.add_argument(
             "--port",
-            type=ArgumentType.integer_range(minimum=1, maximum=2**16),
+            type=ArgumentType.integer_range(minimum=1, maximum=65535),
```

Comment:
Does it make sense to allow HTTP measurements to every possible port?

Reply:
I don't know actually, I guess it's okay? But if we restrict it we should do it on the API side first.